examples/READMED.md

```diff
@@ -0,0 +1,57 @@
+#### 首先我们拉下dubbo rust 项目
+```shell
+git clone https://github.com/apache/dubbo-rust.git
+```
+#### 打开编辑器准备启动项目
+这里我们使用VS code
+打开对应的 **dubbo rust **项目
+在当前目录下创建一个新文件夹，名称为 **.vscode**
+
+```shell
+mkdir .vscode
+```
+
+创建个json文件，名称为launch.json，该文件 **.vscode **目录下 将下文的内容复制到launch.json中
```

Comment:
Can we put the `launch.json` file below into the source code for users to import directly?